MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -18,19 +18,25 @@ final class FeedViewModel {
 		Localized.Feed.title
 	}
 	
+	enum ErrorState {
+		case none
+		case error(String)
+	}
+	
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
-	var onLoadError: Observer<String>?
+	var onErrorStateChange: Observer<ErrorState>?
```

Comment:
Use optional instead - see my other comment.

```suggestion
	var onErrorStateChange: Observer<String?>?
```